src/SpecFuzzPass.cpp

```diff
@@ -588,6 +588,8 @@ auto X86SpecFuzzPass::visitWrite(MachineInstr &MI, MachineBasicBlock &Parent) ->
     // Pushes are a special case as the address is always in RSP
     if (isPush(MI.getOpcode()))
         return visitPush(MI, Parent);
+    if(MI.getOpcode() == X86::MFENCE)
```

Comment:
There is a cleaner way to do it. Just add MFENCE to `X86SpecFuzzPass::isExplicitlySerializing`. In this case, MFENCE will be skipped completely.

Reply:
Makes sense. I'll send an update later.

Reply:
Could you, please, also add a corresponding test to the test suite? See the `tests/` directory for examples.